client-v2/src/actions/Collections.ts

```diff
@@ -48,13 +48,34 @@ function getCollections(collectionIds: string[]): ThunkResult<Promise<void>> {
   return async (dispatch: Dispatch, getState: () => State) => {
     dispatch(collectionActions.fetchStart(collectionIds));
     try {
-      const collections = await fetchCollection(collectionIds);
+      const collectionResponses = await fetchCollection(collectionIds);
       await Promise.all(
-        collections.map(async collection => {
+        collectionResponses.map(async (collectionResponse, index) => {
+          const collectionId = collectionIds[index];
           const collectionConfig = getCollectionConfig(
             getState(),
-            collection.id
+            collectionId
           );
+          if (!collectionResponse) {
```

Comment:
There are a few additional cases to manage here, as any result in an array of returned collections can be null.